atlasdb-perf/src/main/java/com/palantir/atlasdb/performance/benchmarks/table/ConsecutiveNarrowTable.java

```diff
@@ -103,6 +105,26 @@ protected void setupData() {
     }
 
     @State(Scope.Benchmark)
+    public static class RegeneratingCleanNarrowTable extends CleanNarrowTable {
+        @TearDown(Level.Invocation)
+        public void regenerateTable() {
+            getKvs().truncateTable(getTableRef());
+            setupData();
+        }
+
+        @Override
+        public void cleanup() throws Exception {
+            getKvs().dropTable(getTableRef());
+            super.cleanup();
+        }
+
+        @Override
+        public int getNumRows() {
+            return REGENERATING_NUM_ROWS;
+        }
+    }
```

Comment:
Discussed offline - I'd prefer to give this guy his own table ref. Recreating the clean table isn't actually that expensive, but if you run a benchmark using Clean, one using Regenerating-Clean and then one using Clean again, you create the Clean table twice which is unnecessary.